src/login/logind-dbus.c

```diff
@@ -341,6 +341,34 @@ static int property_get_preparing(
         return sd_bus_message_append(reply, "b", b);
 }
 
+static int property_get_preparing_shutdown_with_metadata(
```

Comment:
This has shutdown in its name, si the condition on property below seems unnecessary?